doc/source/netscript/basicfunctions/flags.rst

```diff
@@ -27,14 +32,14 @@ flags() Netscript Function
         tprint(data);
```

Comment:
Well, that's how @hydroflame wrote it; I was just focused on making the output match what it actually looks like if you run the example. You're right, though—the output should be in a comment block instead of the code.